flink-docs/src/main/java/org/apache/flink/docs/rest/RestAPIDocGenerator.java

```diff
@@ -207,44 +207,29 @@ private static String createHtmlEntry(MessageHeaders<?, ?, ?> spec) {
         {
             sb.append("    <tr>\n");
             sb.append("      <td colspan=\"2\">\n");
-            sb.append("      <div class=\"book-expand\">\n");
             sb.append("        <label>\n");
-            sb.append("          <div class=\"book-expand-head flex justify-between\">\n");
-            sb.append("            <span>Request</span>\n");
-            sb.append("            &nbsp;");
-            sb.append("            <span>▾</span>\n");
+            sb.append("          <details>\n");
+            sb.append("          <summary>Request</summary>\n");
```

Comment:
I think these changes accidentally got also committed to this PR.

Reply:
Oh this was intentional since it goes into a similar direction. The deprecated expand macro was essentially exactly what we were doing here for the REST API docs, so I updated that as well so the docs all use the same functionality.